DESCRIPTION

```diff
@@ -35,7 +36,8 @@ Suggests:
     testthat,
     tibble
 Remotes:
-    r-lib/rlang
+    r-lib/rlang,
+    r-lib/ellipsis
```

Comment:
Do you still need the dev version here?

Reply:
I don't see `check_dots_empty()` in the CRAN version.